reference/halyard/high-availability.md

```diff
@@ -186,4 +186,8 @@ With all services enabled for high availability, the new architecture looks like
 
  {% include mermaid %}
 
+## Recoverability
 
+If you've [configured an external Redis](/setup/productionize/caching/externalize-redis/), Spinnaker will be able to recover from failure events. Spot instances on AWS or Preemtible nodes can be used as well to lower cost.
```

Comment:
s/will be able to/can/

Reply:
Thanks @dorbin -- will change that. Does that mean there are further configuration steps needed to allow Spinnaker to recover from failure events?

Reply:
Nah, not that I know of. Just changing four words, future tense, to one word present tense. :)